### ISSUE TITLE
Bulk Edit not updating Status

### ISSUE THREAD

**Issue by davisrael** [grokability/snipe-it#13775]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When selecting multiple assets from a list view across multiple models, status is not updated during bulk edit operations

### Reproduction steps

1. Show a list of assets including multiple models
2. Select **multiple assets** from the list **across multiple models**
3. choose **Edit** from the top left dropdown
4. select **Go**
5. uncheck **Delete values for this asset**
6. change Status
7. press **Save**
...

### Expected behavior

Status of the selected assets is updated

### Snipe-IT Version

v6.2.3 - build 11759 (master)(hosted)

### Operating System

hosted

### Web Server

hosted

### PHP Version

hosted

### Operating System

MacOS Sonoma

### Browser

Google Chrome

### Version

118.0.5993.96 (Official Build) (x86_64)

**Comment by snipe** [comment]
In your tests, are you selecting all of the same status type, or at least the same meta status? Are there some that are checked out and some that are not? If so, I wouldn't expect it to let you change those, since a checkedout status and an archived status (for example) would not be compatible.

**Comment by davisrael** [comment]
All selected assets have the same status of type _pending_ and no meta status (i.e. have not been checked out with a subsequent status change).

**Comment by brynmoorhouse** [comment]
I also had this issue prior to an upgrade to 6.2.3. It appears to have gone away after I've upgraded to 6.2.3 but I'll keep an eye on it as I've only just upgraded and tested it once. 

One thing I noticed was that if I updated status on it's own, with no other updates, the change took effect. If I was updating multiple fields, including status, then status did not update but the other fields did.
